### PR TITLE
Update docs

### LINKED ISSUE #422: Document umode/cmode +u
`+u` was recently introduced as mentioned in the blog post about [ircd-seven 1.1.8](https://freenode.net/news/seven-118) which opts a channel or user out of the server-side spam filtering. this should probably be reflected in [content/kb/using/channelmodes.md](/freenode/web-7.0/blob/master/content/kb/using/channelmodes.md) and [content/kb/using/usermodes.md](/freenode/web-7.0/blob/master/content/kb/using/usermodes.md)

### PR DESCRIPTION
**Pull request by RhinosF1**
Fixes #422

### CODE REVIEW

File: content/kb/using/channelmodes.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ To unset a mode, use `/mode #channel -(mode)`
 | S<br>(SSL-only) | Only users connected via SSL may join the channel while this mode is set. Users already in the channel are not affected. Keep in mind that this also blocks all webchat users, as they are not marked as connected via SSL. |
 | t<br>(ops topic) | Only channel operators may set the channel topic. |
 | z<br>(reduced moderation) | The effects of +b, +q, and +m are relaxed. For each message, if that message would normally be blocked by one of these modes, it is instead sent to channel operators (+o). |
-
+| u<br>(opt out of spam filter)| Messages sent to channel from being subject to server side spam filters. |
```
Comment (svbeon) [inline content/kb/using/channelmodes.md]: I think you accidentally removed a blank line after this

Comment (svbeon) [reply content/kb/using/channelmodes.md]: not a `<br>`(well that too) but in the actuall md file, there was a `\n` before `# Restricted channel modes`

**Comment by Fuchs** [comment]
Probably the wording at #426 is a bit more consistent with the soon-to-be added inbuilt help, but this could be merged for the other changes and then the +u part from #426 could be taken, I guess.

**Comment by svbeon** [comment]
closed by #426

**Pull request closed without merging**